### DISCUSSION TITLE
Checker Help

### DISCUSSION THREAD

**Discussion by deveshjha489** [cpeditor/cpeditor#994] [Q&A] [answered]
How to use checker for testing. I wrote following code for a checker and clicked on add checker and added the file.  When i run my solution code it does not give any result WA or AC.

```
#include "testlib.h"

int main(int argc, char * argv[]) {
    setName("add two signed integers");
    registerTestlibCmd(argc, argv);
    int first  = inf.readInt();
    int second = inf.readInt();
	int res = first + second;
    int pa = ouf.readInt();
    if (res != pa)
        quitf(_wa, "expected %d, found %d", res, pa);
    quitf(_ok, "answer is %d", res);
}
```

**Comment by ouuan** [comment, marked as answer]
You need to enable this option: https://cpeditor.org/docs/preferences/actions/#check-your-answer-on-test-cases-with-empty-output
